addons/sale_project/models/project.py

```diff
@@ -331,8 +331,8 @@ def _get_profitability_labels(self):
         return {
             **super()._get_profitability_labels(),
             'service_revenues': _lt('Other Services'),
-            'other_revenues': _lt('Materials'),
-            'other_invoice_revenues': _lt('Other Revenues'),
+            'other_revenues': _lt('Other Revenues'),
+            'other_invoice_revenues': _lt('Other Invoice Revenues'),
```

Comment:
I would let the previous label here, to me, the bug is more the label in `timesheet_invoice_type` field as you did below. 🙂

Reply:
Alright, I'll stick to timesheet_invoice_type then